packages/block-editor/src/components/list-view/use-list-view-drop-zone.js

```diff
@@ -179,26 +180,39 @@ function getListViewDropTarget( blocksData, position ) {
 /**
  * A react hook for implementing a drop zone in list view.
  *
+ * @param {string} dropZoneName the name of the drop zone.
  * @return {WPListViewDropZoneTarget} The drop target.
  */
-export default function useListViewDropZone() {
+export default function useListViewDropZone( dropZoneName ) {
 	const {
 		getBlockRootClientId,
 		getBlockIndex,
 		getBlockCount,
 		getDraggedBlockClientIds,
 		canInsertBlocks,
-	} = useSelect( blockEditorStore );
+		getDraggedBlocksTargets,
+	} = unlock( useSelect( blockEditorStore ) );
 	const [ target, setTarget ] = useState();
 	const { rootClientId: targetRootClientId, blockIndex: targetBlockIndex } =
```

Comment:
it's a bit confusing the difference between the `target` variable this hook returns and the target finding done later.